src/views/Timer.vue

```diff
@@ -17,6 +17,7 @@
         <sortie v-if="componentState.sortie.display" :sortie="worldstate.sortie" />
         <arbitration v-if="componentState.arbitration.display" :arbitration="worldstate.arbitration" />
         <fissures v-if="componentState.fissures.display" :fissures="worldstate.fissures" />
+        <kuvas v-if="componentState.kuvas.display" :kuvas="worldstate.kuva" />
```

Comment:
while we're in here, should we move kuva & arbitrations up towards the top since they're some of the most used...

so like 
- cycle timers
- kuva
- arbitrations
- event
- news
-sortie

then the rest as-is?